pkg/intervalpolicy/interval_policy.go

```diff
@@ -0,0 +1,10 @@
+package intervalpolicy
+
+import "time"
+
+// IntervalPolicy defines a policy to return interval based on the received events.
+type IntervalPolicy interface {
+	Evaluate()
```

Comment:
can you add documentation for the interface functions?
e.g. 
```
// Reset resets the interval policy
// GetInterval returns current interval
...
```

Reply:
done